views/start.html

```diff
@@ -0,0 +1,64 @@
+{% extends "layout.html" %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+
+{% block pageTitle %}
+ {{ SERVICE_NAME }} - GOV.UK
+{% endblock %}
+
+<!-- No service name header-->
+{% block header %}
+  {{ govukHeader({
+    homepageUrl: "https://www.gov.uk",
+    containerClasses: "govuk-width-container"})
+}}
+{% endblock %}
+
+{% block signoutBar %}
+{# Remove signout bar on this page by replacing it with nothing #}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-xl">
+        Remove a company director
+      </h1>
+
+      {% if FEATURE_FLAG_REMOVE_DIRECTOR_20022023 === "true" %}
+        <p class="govuk-body">Use this service to remove (terminate the appointment of) a director of a limited company on the Companies House register, including corporate directors.</p>
```

Comment:
I guess this is placeholder test for Harry's story? We could clean it up though, remove the feature flag etc...